src/main/java/org/wildfly/security/auth/client/AuthenticationConfiguration.java

```diff
@@ -474,6 +475,54 @@ public AuthenticationConfiguration useChoice(String choice) {
     }
 
     /**
+     * Create a new configuration which is the same as this configuration, but which uses the given one-time password
+     * to authenticate.
+     *
+     * @param password the one-time password to use in either hex or multi-word format
+     * @return the new configuration
+     */
+    public AuthenticationConfiguration useOneTimePassword(String password) {
```

Comment:
Is there any way this could be done using a credential instead so that we don't need a method specifically for OTP?